pulp_ansible/app/galaxy/views.py

```diff
@@ -55,7 +56,7 @@ def _distro_content(self):
 
     def get_serializer_context(self):
         """Inserts distribution path to a serializer context."""
-        context = super().get_serializer_context()
+        context = {}
```

Comment:
I didn't go too in-depth here, but at some point I was getting an `AttributeError` that `super()` didn't have an attribute `get_serializer_context`.  Since this was never used prior to now, and an empty dict worked, I just swapped it out.

Reply:
I think we should still call it. It is a method from drf. And not calling super on it may break in unforseen ways. When you get AttributeError, can you tell for which class? Maybe the order of mixins is wrong there.

Reply:
I will see if I can replicate and relay that information.

Reply:
Ok, solved this (I think), it was due to not using the `generics.GenericAPIView` and instead using `views.APIView1`